src/js/modules/sections/components/SectionPage.js

```diff
@@ -150,18 +138,22 @@ const SectionPage = ({ classes, articles, subsections, section, match, featuredM
     });
   };
   return (
-    <div className={classes.SectionPage}>
+    <div className={classes.entireSectionPage}>
       <h1 className={classes.sectionName}>{section.name}</h1>
       {/* <p>description: {section.description}</p> */}
       <ul className={classes.subsectionLink}>
         {createSubsectionLinks()}
       </ul>
       <hr className={classes.sectionNameDivider}/>
-      <Grid className={classes.articleList}>
-        {createArticleLinks()}
-      </Grid>
     </div>
   );
+    const ArticleList = ({ articles, featuredMedia }) => {
+        return (
+          <Grid className={classes.articleList}>
+            {createArticleLinks()}
+          </Grid>
+      );
+    };
 };
```

Comment:
This is all correct, but ArticleList should be its own component. That means you have to make it its own file. In your current code, ArticleList is not even rendered because it is not in the return statement (you do this by including the line `<ArticleList/>`, and then passing articles and featuredMedia into its props.

If `ArticleList` is moving into its own file, then `createArticleLinks()` has to go with it.